app/components/navs/nav-bar.tsx

```
@@ -45,26 +45,32 @@ export default function NavBar() {
+      router.push('/search');
     }
   };
 
+  const handleLoginClick = () => {
+    const currentPath = pathname;
+    const loginPath = `/login?returnUrl=${encodeURIComponent(currentPath)}`;
+    router.push(loginPath);
+  };
+
   return (
     <div className="nav-bar flex flex-grow-0 pl-4 md:pl-0 pt-3 items-center justify-between w-full top-0 max-h-24 sticky bg-inherit z-[1000]" autoFocus={true}>
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)